src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/Management/WebService.cs

```diff
@@ -0,0 +1,273 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Management.MachineLearning.WebServices;
+using Microsoft.Azure.Management.MachineLearning.WebServices.Models;
+using Microsoft.Azure.Management.MachineLearning.WebServices.Util;
+
+namespace Microsoft.Azure.MachineLearning
+{
+
+    /// <summary>
+    /// Describes an AzureML Web Service.
+    /// </summary>
+    public class WebService
+    {
+        /// <summary>
+        /// Internal reference to a AzureMLWebServicesManagementClient.
+        /// </summary>
+        private AzureMLWebServicesManagementClient _client { get; set; }
+
+        /// <summary>
+        /// Internal autogenerated model that holds properties of a web service.
+        /// TODO: Call it webservicedefinition in the swagger
+        /// </summary>
+        internal Management.MachineLearning.WebServices.Models.WebService Definition { get; set; }
+
+        /// <summary>
+        /// The name of this web service.
+        /// </summary>
+        public string Title
+        {
+            get { return this.Definition.Properties.Title; }
+            set
+            {
+                this.Definition.Properties.Title = value;
+                this.Definition.Name = value;
+            }
+        }
+
+        /// <summary>
+        /// The description for this web service.
+        /// </summary>
+        public string Description
+        {
+            get { return this.Definition.Properties.Description; }
+            set { this.Definition.Properties.Description = value; }
+        }
+
+        /// <summary>
+        /// DateTime for when this web service was created. Is readonly.
+        /// </summary>
+        public DateTime? CreatedOn
+        {
+            get
+            {
+                try
+                {
+                    var val = this.Definition.Properties.CreatedOn.Value;
+
+                    return val;
+                }
+                catch (Exception)
```

Comment:
catch exception in a getter seems weird. Can you just return Properties.CreatedOn? Its type is also "DateTime?".